physics/GWD/ugwpv1_gsldrag.F90

```diff
@@ -1,7 +1,5 @@
 !>  \file ugwpv1_gsldrag.F90
-
-!> This module introduces two gravity wave drag schemes: UGWPv1 and orographic drag scheme 
-!!
+!! This introduces two gravity wave drag schemes ugwpv1/CIRES and GSL/drag_suite.F90 under "ugwpv1_gsldrag" suite:
```

Comment:
@mzhangw There are several changes in this file that affect the scientific documentation. Could you please review them? My initial impression is that these changes should be reverted.